python/py-pylru/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pylru
+version             1.1.0
+revision            0
```

Comment:
FYI, we don't do `revision 0` in Portfiles, we only add the line once revision hits 1.